dio/lib/src/dio_event_processor.dart

```diff
@@ -0,0 +1,110 @@
+// ignore_for_file: implementation_imports, public_member_api_docs
+
+import 'dart:async';
+
+import 'package:dio/dio.dart';
+import 'package:sentry/sentry.dart';
+import 'package:sentry/src/sentry_exception_factory.dart';
+import 'package:sentry/src/sentry_stack_trace_factory.dart';
+
+class DioEventProcessor extends EventProcessor {
```

Comment:
```suggestion
class DioEventProcessor implements EventProcessor {
```

Reply:
Does that make any difference here?

Reply:
Well it is an `interface`, but I am coming from the Java world so when I see `extends`, I expect some implementation to exist in the superclass. Not really important, up to you :)